scripts/docker_entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Double compilation to get correct references
+pdflatex -interaction=nonstopmode "$(basename "$1")"
```

Comment:
Once inside a function, we generally like to pull out the positional arguments into local variables for clearer reading.
E.g.,:
```
function main() {
    local path="$1"

    ...
}
```